### PR TITLE
Update Mo2 to V2.5

### LINKED ISSUE #362: Skyrim Together Reborn Support
Using MO2 to launch Skyrim together reborn does not work as it will give the the error
image

To install Skyrim Together Reborn i followed the official guide
https://wiki.tiltedphoques.com/tilted-online/guides/client-setup/skyrim-together-reborn

I have tested and launched it straight via steam and that worked so this is an issue only in MO2 Linux

errors i see in the terminal when launcher STR are:
``EnableNonClientDpiScaling() failed for HWND 0x3011a (120) (Call not implemented.)``
``[2022-07-14 12:45:24.058 W] [creationgameplugins.cpp:133] Z:/home/niki/Games/mod-organizer-2-skyrimspecialedition/modorganizer2/profiles/Skyrim Together Reborn/plugins.txt empty``

### PR DESCRIPTION
**Pull request by GamzeeRakoon**
Proton 9.0 is now stable and can be used to launch Mo2 v2.5!

I have tested and it runs perfectly for me

can confirm this fixes #362

### CODE REVIEW

File: step/download_external_resources.sh [hunk 1]
```diff
@@ -3,9 +3,9 @@
 download="$utils/download.sh"
 extract="$utils/extract.sh"
 
-jdk_url='https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u312-b07/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip'
+jdk_url='https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u412-b08/OpenJDK8U-jre_x86-32_windows_hotspot_8u412b08.zip'
```
Comment (rockerbacon) [inline step/download_external_resources.sh:6]: JDK is finicky, I tested a few different builds and tools and found this version to be the least problematic. It's better to do this update in a separate PR, with a bit more testing and discussion on what it breaks and fixes.

**Review by rockerbacon — CHANGES_REQUESTED** [review]

File: step/download_external_resources.sh (hunk 1 above)
Comment (GamzeeRakoon) [reply step/download_external_resources.sh]: sounds fair reverted the change!

**Comment by rockerbacon** [comment]
Thank you for your contribution!
